tests/tests.py

```diff
@@ -607,6 +606,12 @@ def test_312(self):
     def test_316(self):
         Category.objects.cache().annotate(num=Count('posts')).aggregate(total=Sum('num'))
 
+    def test_333(self):
+        # ensure that we can use a manager not derived from Manger
+        # (i.e. using manager.BaseManager.from_queryset)
+        mgr = CustomFromQSModel.objects
+        assert isinstance(mgr, CustomFromQSManager)
```

Comment:
This doesn't really test anything. The old code will blow up in `install_cacheops()` call I suppose. You may just drop it.